dashboard/app/models/lesson_group.rb

```diff
@@ -20,7 +20,9 @@ class LessonGroup < ApplicationRecord
   include SerializedProperties
 
   belongs_to :script
-  has_many :lessons, -> {order('absolute_position ASC')}
+  has_many :lessons, -> {order(:absolute_position)}, dependent: :destroy, class_name: 'Lesson'
```

Comment:
can omit `class_name: 'Lesson'`? I may be mistaken, but I would think this would be implied by the name `:lessons`